tests/test_analyze_common.py

```diff
@@ -32,7 +32,7 @@ def tearDown(self):
         del self.test_dockerfile
 
     def testGetShellCommands(self):
-        command = common.get_shell_commands("yum install nfs-utils")
+        command, _ = common.get_shell_commands("yum install nfs-utils")
```

Comment:
If you can add a test here for a branching command such as `if [ -z $var ]; then do something; fi`